app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
+# Helper methods
 module ApplicationHelper
-end
+  def login_helper (style)
+    if current_user.is_a?(User)
+      link_to "Logout", destroy_user_session_path, method: :delete, class: style
```

Comment:
@sohilp393, Prefer single-quoted strings when you don't need string interpolation.

Reply:
Removed double-quoted strings and rubocop warnings